data/mods/Magiclysm/Spells/attunements/Shaman.json

```
@@ -0,0 +1,67 @@
+[
+  {
+    "id": "butterfly_swarm",
+    "type": "SPELL",
+    "name": { "str": "Butterfly Swarm" },
+    "description": "Summon pack of butterflies with razor-sharp wings, to cut every living creature on your way.",
+    "valid_targets": [ "ground" ],
+    "flags": [ "SOMATIC", "VERBAL", "CONCENTRATE" ],
+    "effect": "summon",
+    "effect_str": "mon_razor_butterflies",
+    "shape": "blast",
+    "spell_class": "SHAMAN",
+    "energy_source": "MANA",
+    "difficulty": 7,
+    "max_level": 35,
+    "base_casting_time": 500,
+    "casting_time_increment": -10,
+    "final_casting_time": 150,
+    "base_energy_cost": 350,
+    "energy_increment": -5,
+    "final_energy_cost": 200,
+    "min_damage": 1,
+    "max_damage": 12,
+    "damage_increment": 0.4,
+    "min_range": 0,
+    "max_range": 8,
+    "range_increment": 0.25,
+    "min_aoe": 1,
+    "max_aoe": 8,
+    "aoe_increment": 0.2,
+    "min_duration": 14400,
+    "max_duration": 144000,
+    "duration_increment": 4000
+  },
+  {
+    "id": "summon_shambling_mound",
+    "type": "SPELL",
+    "name": { "str": "Call of Mound" },
+    "description": "Summon the Shambling Mound, the giant carnivorous plant mess with devastating area attacks.  It is too slow for common fight, but it is the best defender for skilled druid.",
```

droseras r sundews, basically a tentacle thing
```suggestion
    "description": "Summon a Shambling Mound, a giant Drosera with devastating area attacks.  It is very slow, but is an excellent defender.",
```

---

It doesn't looks like drosera, just google some images, its from DnD